packages/next-templates/src/app/meldingen-openbare-ruimte/graffiti-melding/form/page.tsx

```diff
@@ -177,6 +175,7 @@ export default function Home() {
                     minLength: { value: 4, message: 'Min length is 4' },
                   })}
                   placeholder=""
+                  name="Straat"
```

Comment:
name wilt je in het engels want die naam gebruikt de backend. dus je wilt het consistent houden omdat de id en classname ook engels zijn.